pkgs/applications/misc/systembus-notify/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, systemd }:
+
+stdenv.mkDerivation rec {
+  pname = "systembus-notify";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "rfjakob";
+    repo = "systembus-notify";
+    rev = "v${version}";
+    sha256 = "11zq84qfmbyl51d3r6294l2bjhlgwa9bx7d263g9fkqrwsg0si0y";
+  };
+
+  buildInputs = [ systemd ];
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 systembus-notify -t $out/bin
+    install -Dm644 systembus-notify.desktop -t $out/share/applications
```

Comment:
This should go in `$out/etc/xdg/autostart/`

Reply:
If so, is there any way to disable it manually?
And I'm not sure whether `$out/etc/xdg/autostart` will work for user installation through `nix-env -i`.

Reply:
nix-env should work provided you have $HOME/.nix-profile/etc/xdg in XDG_CONFIG_DIRS. It's pretty standard for apps to do this (see /run/current-system/sw/etc/xdg/autostart for some examples). This also matches what is in the `install` rule of the Makefile (although it references $HOME).